2024-02-28 11:38:35,150 INFO    MainThread:2396 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-02-28 11:38:35,150 INFO    MainThread:2396 [wandb_setup.py:_flush():76] Configure stats pid to 2396
2024-02-28 11:38:35,150 INFO    MainThread:2396 [wandb_setup.py:_flush():76] Loading settings from C:\Users\CYP\.config\wandb\settings
2024-02-28 11:38:35,150 INFO    MainThread:2396 [wandb_setup.py:_flush():76] Loading settings from c:\Users\CYP\Documents\GitHub\AIAKI\wandb\settings
2024-02-28 11:38:35,150 INFO    MainThread:2396 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-28 11:38:35,150 INFO    MainThread:2396 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-02-28 11:38:35,152 INFO    MainThread:2396 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-28 11:38:35,152 INFO    MainThread:2396 [wandb_init.py:_log_setup():526] Logging user logs to c:\Users\CYP\Documents\GitHub\AIAKI\wandb\run-20240228_113835-vse5gidb\logs\debug.log
2024-02-28 11:38:35,152 INFO    MainThread:2396 [wandb_init.py:_log_setup():527] Logging internal logs to c:\Users\CYP\Documents\GitHub\AIAKI\wandb\run-20240228_113835-vse5gidb\logs\debug-internal.log
2024-02-28 11:38:35,152 INFO    MainThread:2396 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001FF252B71F0>
2024-02-28 11:38:35,152 INFO    MainThread:2396 [wandb_init.py:init():566] calling init triggers
2024-02-28 11:38:35,152 INFO    MainThread:2396 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.0005, 'batch_size': 128, 'n_units': 64}
2024-02-28 11:38:35,152 INFO    MainThread:2396 [wandb_init.py:init():616] starting backend
2024-02-28 11:38:35,152 INFO    MainThread:2396 [wandb_init.py:init():620] setting up manager
2024-02-28 11:38:35,153 INFO    MainThread:2396 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-02-28 11:38:35,155 INFO    MainThread:2396 [wandb_init.py:init():628] backend started and connected
2024-02-28 11:38:35,158 INFO    MainThread:2396 [wandb_run.py:_label_probe_notebook():1295] probe notebook
2024-02-28 11:38:35,159 INFO    MainThread:2396 [wandb_run.py:_label_probe_notebook():1305] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-02-28 11:38:35,159 INFO    MainThread:2396 [wandb_init.py:init():720] updated telemetry
2024-02-28 11:38:35,330 INFO    MainThread:2396 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-02-28 11:38:35,800 INFO    MainThread:2396 [wandb_run.py:_on_init():2262] communicating current version
2024-02-28 11:38:35,845 INFO    MainThread:2396 [wandb_run.py:_on_init():2271] got version response 
2024-02-28 11:38:35,845 INFO    MainThread:2396 [wandb_init.py:init():804] starting run threads in backend
2024-02-28 11:38:35,971 INFO    MainThread:2396 [wandb_run.py:_console_start():2241] atexit reg
2024-02-28 11:38:35,971 INFO    MainThread:2396 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2024-02-28 11:38:35,971 INFO    MainThread:2396 [wandb_run.py:_redirect():2161] Wrapping output streams.
2024-02-28 11:38:35,971 INFO    MainThread:2396 [wandb_run.py:_redirect():2186] Redirects installed.
2024-02-28 11:38:35,971 INFO    MainThread:2396 [wandb_init.py:init():847] run started, returning control to user process
2024-02-28 12:47:01,506 INFO    MainThread:2396 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-28 12:47:01,506 INFO    MainThread:2396 [wandb_init.py:_pause_backend():437] pausing backend
2024-02-28 12:47:13,186 INFO    MainThread:2396 [wandb_init.py:_resume_backend():442] resuming backend
2024-02-28 12:47:13,187 INFO    MainThread:2396 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-28 12:47:13,187 INFO    MainThread:2396 [wandb_init.py:_pause_backend():437] pausing backend
2024-02-28 12:47:13,388 INFO    MainThread:2396 [wandb_init.py:_resume_backend():442] resuming backend
2024-02-28 12:47:13,391 INFO    MainThread:2396 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-28 12:47:13,391 INFO    MainThread:2396 [wandb_init.py:_pause_backend():437] pausing backend
2024-02-28 12:47:13,555 INFO    MainThread:2396 [wandb_init.py:_resume_backend():442] resuming backend
2024-02-28 12:47:13,564 INFO    MainThread:2396 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-02-28 12:47:13,564 INFO    MainThread:2396 [wandb_setup.py:_flush():76] Configure stats pid to 2396
2024-02-28 12:47:13,564 INFO    MainThread:2396 [wandb_setup.py:_flush():76] Loading settings from C:\Users\CYP\.config\wandb\settings
2024-02-28 12:47:13,564 INFO    MainThread:2396 [wandb_setup.py:_flush():76] Loading settings from c:\Users\CYP\Documents\GitHub\AIAKI\wandb\settings
2024-02-28 12:47:13,564 INFO    MainThread:2396 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-28 12:47:13,564 INFO    MainThread:2396 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-02-28 12:47:13,565 INFO    MainThread:2396 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-28 12:47:13,565 INFO    MainThread:2396 [wandb_init.py:_log_setup():526] Logging user logs to c:\Users\CYP\Documents\GitHub\AIAKI\wandb\run-20240228_124713-7tcbl4yi\logs\debug.log
2024-02-28 12:47:13,565 INFO    MainThread:2396 [wandb_init.py:_log_setup():527] Logging internal logs to c:\Users\CYP\Documents\GitHub\AIAKI\wandb\run-20240228_124713-7tcbl4yi\logs\debug-internal.log
2024-02-28 12:47:13,565 INFO    MainThread:2396 [wandb_init.py:init():566] calling init triggers
2024-02-28 12:47:13,565 INFO    MainThread:2396 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.0005, 'batch_size': 128, 'n_units': 8}
2024-02-28 12:47:13,565 INFO    MainThread:2396 [wandb_init.py:init():591] re-initializing run, found existing run on stack: vse5gidb
2024-02-28 12:47:13,566 INFO    MainThread:2396 [wandb_run.py:_finish():1970] finishing run dahs/IMV_LSTM_4hrs_mean/vse5gidb
2024-02-28 12:47:13,566 ERROR   MainThread:2396 [jupyter.py:save_history():437] Run pip install nbformat to save notebook history
2024-02-28 12:47:13,566 INFO    MainThread:2396 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-28 12:47:13,566 INFO    MainThread:2396 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-02-28 12:47:13,567 INFO    MainThread:2396 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-02-28 12:47:13,567 INFO    MainThread:2396 [wandb_run.py:_restore():2193] restore
2024-02-28 12:47:13,567 INFO    MainThread:2396 [wandb_run.py:_restore():2199] restore done
2024-02-28 12:47:18,162 INFO    MainThread:2396 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-02-28 12:47:18,166 INFO    MainThread:2396 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-02-28 12:47:18,170 INFO    MainThread:2396 [wandb_run.py:_footer_sync_info():3825] logging synced files
